root/components/EntityTabs.js

```diff
@@ -124,6 +124,13 @@ function buildLinks(
     links.push(buildLink(l('Tags'), entity, 'tags', page));
   }
 
+  if (entityProperties.ratings || entityProperties.reviews) {
+    const ratingsTabTitle = entityProperties.reviews
+      ? l('Reviews')
+      : l('Ratings');
```

Comment:
It'd be pretty cool if we displayed the number of ratings/reviews in parentheses here like we do for disc IDs.

Reply:
I did think about it. But that means hitting CB for the amount of reviews with every page load, is this something we worry about at all?

Also, not sure how to show *both* the number of reviews and ratings if both exist. Should we even try? It might be confusing to show Reviews (0) if there's a lot of ratings.

Reply:
> But that means hitting CB for the amount of reviews with every page load, is this something we worry about at all?

True, guess I was thinking they were already cached there. That might be something we could load asynchronously in the browser.

> Also, not sure how to show both the number of reviews and ratings if both exist. Should we even try? It might be confusing to show Reviews (0) if there's a lot of ratings.

This is tricky. We could revert to "Ratings" if there are ratings but no reviews (though I guess "Reviews" helps indicate that it's reviewable). Or we could change the tab name to "Ratings and Reviews" if both are supported and display a total count. Or something else.

Needs more thought, so maybe not in this PR. I still like the idea of indicating if there's content though, to save people from wasting a click.

Reply:
> Ratings and Reviews

That seems too long, that's why I didn't do that.

> I still like the idea of indicating if there's content though, to save people from wasting a click.

I generally agree, although hey, maybe they'll want to write a review if it's empty? :p

Reply:
I'd merge as-is for now, but if https://tickets.metabrainz.org/browse/CB-427 is implemented as mentioned then we could call that (and maybe cache it) and get all the data in one call rather than 2/3.

Reply:
Also, all edits support reviews *and* ratings now AFAICT, since Recording is now supported in CB too. But maybe it makes sense to keep this conditional here in case we end up getting more entities that can be rated but not reviewed?